### PR TITLE
chore(L10n): adding missing files

### PR DESCRIPTION
**Pull request by nahumzs**
### Purpose 🚀
adding missing l10n files.

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [ ] MINOR (backward compatible) change to _these packages_
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/L10n/src/locales-js/de.js [hunk 1]
```diff
@@ -70,7 +73,7 @@ const locales = {
         "arrowdown": "Ab",
         "escape": "Escape"
       },
-      "externalLink": {
+      "externaLink": {
```
Comment (oscarkwan) [inline packages/L10n/src/locales-js/de.js]: why changed this?

Comment (nahumzs) [reply packages/L10n/src/locales-js/de.js]: I didnt the script did

**Comment by nahumzs** [comment]
ohhh I think is an error by the l10n team https://github.com/acl-services/paprika/pull/328/files#diff-cb8123a2d323f810ca362e2c4b695b19L59

**Comment by nahumzs** [comment]
will be fix by L10n team cc @alexandermccate 🙇

**Comment by alexandermccate** [comment]
@nahumzs @oscarkwan The key was changed earlier in EN resource file. Please fix it when you have a chance. 

https://github.com/acl-services/paprika/blob/02878656cf2acd9c62af07d4123dced3ab4455eb/packages/L10n/src/locales/en.yml#L61

**Comment by tristanjasper** [comment]
> @nahumzs @oscarkwan The key was changed earlier in EN resource file. Please fix it when you have a chance.
> 
> https://github.com/acl-services/paprika/blob/02878656cf2acd9c62af07d4123dced3ab4455eb/packages/L10n/src/locales/en.yml#L61

Fixed in https://github.com/acl-services/paprika/pull/334

**Pull request closed without merging**